README.rst

```diff
@@ -36,3 +36,11 @@ Manages /etc/letsencrypt/cli.ini config file.
 -----------
 Creates a certificate with the domains in each domain set (letsencrypt:domainsets in pillar). Letsencrypt uses a relatively short validity of 90 days.
 Therefore, a cron job for automatic renewal every 60 days is installed for each domain set as well.
+
+``letsencrypt.dhparam``
+-----------
+Creates either a global dhparam.pem in /etc/letsencrypt/live/dhparam.pem or one dhparam per domainset in /etc/letsencrypt/live/<first-domainname>/dhparam.pem.
+
+To enable it you have to set **letsencrypt.dhparam.enable** to **True** and for global dhparams you need to set **letsencrypt.dhparam.global_dhparam** to **False**.
```

Comment:
This sentence is a pretty long line and I don't think it's correct. `for per domainset dhparams you need to set ... to **False**`